authui/src/authflowv2/icons/authgear-icon.css

```diff
@@ -0,0 +1,20 @@
+.authgear-icon {
+  @apply w-16 h-16 bg-no-repeat bg-center;
+  background-image: url(./icons/raw/icon-authgear.svg);
+
+  :root[alignment-logo="start"] & {
+    @apply mr-auto ml-0;
+  }
+
+  :root[alignment-logo="center"] & {
+    @apply mr-auto ml-auto;
+  }
+
+  :root[alignment-logo="end"] & {
+    @apply mr-0 ml-auto;
+  }
+
+  :root[alignment-logo="hidden"] & {
+    @apply hidden;
+  }
```

Comment:
Alignment depends on the parent container. Here we assume the parent container is a block. If it is a flex container, we need different sets of style to position to logo. Let's keep this for now. We need to revisit this when we actually implement the logo header.